website/src/css/custom.css

```diff
@@ -1,6 +1,35 @@
 /* Stick any global custom CSS in this file */
 
 /* You can override the default Infima variables here. */
+@layer theme, base, components, utilities;
+@import "tailwindcss/theme.css" layer(theme);
+@import "tailwindcss/utilities.css" layer(utilities);
+
+@config "../../tailwind.config.js";
+
+/*
+  The default border color has changed to `currentColor` in Tailwind CSS v4,
+  so we've added these compatibility styles to make sure everything still
+  looks the same as it did with Tailwind CSS v3.
+
+  If we ever want to remove these styles, we need to add an explicit border
+  color utility to any element that depends on these defaults.
+*/
+@layer base {
+
+  *,
+  ::after,
+  ::before,
+  ::backdrop,
+  ::file-selector-button {
+    border-color: var(--color-gray-200, currentColor);
```

Comment:
It was added by `@tailwindcss/upgrade` automatically.

https://tailwindcss.com/docs/upgrade-guide#default-border-color